sdk/storage/azure-storage-blobs/swagger/README.md

```diff
@@ -688,13 +709,23 @@ directive:
 
 ```yaml
 directive:
+  - from: swagger-document
+    where: $["x-ms-paths"]["/?comp=blobs"].get.parameters
+    transform: >
+      $ = $.filter(p => !p["$ref"] || !p["$ref"].endsWith("#/parameters/FilterBlobsInclude"));
+  - from: swagger-document
+    where: $["x-ms-paths"]["/{containerName}?restype=container&comp=blobs"].get.parameters
+    transform: >
+      $ = $.filter(p => !p["$ref"] || !p["$ref"].endsWith("#/parameters/FilterBlobsInclude"));
   - from: swagger-document
     where: $.definitions
     transform: >
       $.FilterBlobItem["x-ms-client-name"] = "TaggedBlobItem";
       $.FilterBlobItem.properties["Name"]["x-ms-client-name"] = "BlobName";
       $.FilterBlobItem.properties["ContainerName"]["x-ms-client-name"] = "BlobContainerName";
       delete $.FilterBlobItem.properties["TagValue"];
+      delete $.FilterBlobItem.properties["VersionId"];
+      delete $.FilterBlobItem.properties["IsCurrentVersion"];
```

Comment:
Why is this feature deleted here? Is this tracked anywhere?

Reply:
Because this feature is not in stg85/86 even stg87. And I have asked Emma. It's also deleted in JS. 
I have tracked in my OneNote. And is there any issue to track it ?

Reply:
It's fine.